Documentation/gettingstarted/docker.rst

```diff
@@ -96,9 +96,10 @@ it using the ``cilium`` CLI client. Check the status of the agent by running
     Cilium health daemon:   Ok
     IPAM:                   IPv4: 2/65535 allocated from 10.15.0.0/16,
     Controller Status:      14/14 healthy
-    Proxy Status:           OK, ip 10.15.225.211, 0 redirects active on ports 10000-20000
-    Hubble:                 Disabled
-    Cluster health:         1/1 reachable   (2020-04-17T10:55:03Z)
+    Proxy Status:           OK, ip 10.15.66.191, 0 redirects active on ports 10000-20000
+    Cluster health:         0/1 reachable   (2020-10-22T09:54:59Z)
+      Name                  IP              Reachable   Endpoints reachable
+      vagrant (localhost)   10.0.2.15       true        false
```

Comment:
It should become `reachable` after a while.

Reply:
Indeed, and look something like:
```
Cluster health:         4/4 reachable   (2020-10-22T10:59:19Z)
```
Also, it looks like Hubble status was removed. With 1.9 enabling Hubble by default, it should actually look something like this:
```
Hubble:                 Ok              Current/Max Flows: 4096/4096 (100.0%), Flows/s: 1.29   Metrics: Disabled
```

Reply:
Correct, I should have waited a few seconds more to see that endpoint in a `reachable` state

Reply:
This made me realize that the `Vagrantfile` for that tutorial is still using a docker image for cilium based on version `1.7`.
I bumped it [in this PR](https://github.com/cilium/cilium/pull/13697)

Reply:
@jibi How come Hubble is marked as "Disabled"? It should be enabled by default as stated in my comment above.

Reply:
I suspect it's because we are not pulling the hubble images in the [docker-compose.yml](https://github.com/cilium/cilium/blob/master/examples/getting-started/docker-compose.yml) file (although I have still not much context on how the docker setup works for hubble)

Reply:
Starting with Cilium 1.8, Hubble server is embedded in Cilium so not extra image is required. Actually, the reason why it does not show up is because the Cilium version is 1.7.2 as you found out in #13697. I think that the command output should be updated for v1.9.

Reply:
> I think that the command output should be updated for v1.9.

I actually had already updated the image before running that command :thinking: 
```sh
vagrant@vagrant:~/cilium/examples/getting-started$ cilium version
Client: 1.9.0-rc2 fae41eead 2020-10-21T21:03:17+11:00 go version go1.15.3 linux/amd64
Daemon: 1.9.0-rc2 fae41eead 2020-10-21T21:03:17+11:00 go version go1.15.3 linux/amd64
vagrant@vagrant:~/cilium/examples/getting-started$ cilium status | grep Hubble
Hubble:                 Disabled
```

Reply:
Mmh, not sure why Hubble is disabled then :thinking: .

Reply:
Is it worth opening an issue? Sounds like Docker integration is best effort (no large users).